x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -75,10 +77,12 @@ export const SecuritySolutionTemplateWrapper: React.FC<SecuritySolutionPageWrapp
       getTimelineShowStatus(state, TimelineId.active)
     );
 
+    const { indicesExist } = useSourcererScope();
```

Comment:
@michaelolo24 @jonathan-buttner - to get the tests to pass, I believe this `indicesExist` needs to resolve to true when the `metrics-endpoint.metadata` index exists in addition to any of the other indices that this already tracks.

The issue is the new empty state covers the entire app, including the Endpoint management page which tracks just Endpoint metadata.

I'm happy to help make the change, just wanted to point this out.  I can work with you offline.

Reply:
@kevinlog yea, that might be a good idea. I know @stephmilovic is making updates to a lot of the sourcerer stuff now, so might be good to sync up with her as well

Reply:
I spoke with @stephmilovic offline - a potential solution is to use the `useFetchIndex` hook in combination with this check in order to check for the existence of `metrics-endpoint.metadata`.  I can take a closer look at this a bit later.

Reply:
Great, thanks Kevin!